base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -379,7 +403,102 @@ impl LMDBDatabase {
         Ok(())
     }
 
-    fn op_update_block_accumulated_data(
+    fn insert_block_body(
+        &mut self,
+        txn: &WriteTransaction<'_>,
+        header_hash: HashOutput,
+        body: AggregateBody,
+    ) -> Result<(), ChainStorageError>
+    {
+        let (inputs, outputs, kernels) = body.dissolve();
+
+        let BlockAccumulatedData {
```

Comment:
This  struct also does not make sense as a default. The values should be derived from a header or not exist at all.